src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -234,23 +235,26 @@ public void Initialize(bool generateNewDatabaseId = false)
             {
                 Initialize(options);
             }
-            catch (Exception)
+            catch (Exception e)
             {
                 options.Dispose();
+
+                if (DocumentDatabase.Configuration.Core.RunInMemory == false && e.Message.StartsWith("No such journal", StringComparison.Ordinal))
```

Comment:
Create "No such journal" constant and use it here and when throwing the exception